mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.java

```diff
@@ -737,9 +763,10 @@ public void onReceivedError(WebView view, int errorCode, String description,
                     updateViewVisibility(true, false);
                 }
             });
-            webView.getSettings().setDomStorageEnabled(true);
-            webView.getSettings().setJavaScriptEnabled(true);
-            webView.loadUrl(url);
+            mWebview.getSettings().setDomStorageEnabled(true);
+            mWebview.getSettings().setJavaScriptEnabled(true);
+            mWebview.loadUrl(url);
+            mWebview.setBackgroundColor(Color.TRANSPARENT);
```

Comment:
This should be possible to do in XML as well (`android:background=@null` I think)